.github/workflows/pull-request.yml

```
@@ -0,0 +1,500 @@
+        if: github.event.pull_request.base.ref != 'develop-postgres'
+        run: |
+          echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
+          exit 1
+
+  CodeRabbit-Approval:
+    name: Validate CodeRabbit.ai Approval
+    needs: [Code-Quality-Checks, Docker-Start-Check, Start-App-Without-Docker, Test-Application]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check CodeRabbit.ai Approval
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2181:style:4:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)